conduct.md

```diff
@@ -39,31 +39,39 @@ The deadline for each task should be 12am. This will help other contributors to
 
 ### Communication
 
-TODO: Write an introductory paragraph.
+Communication is the cornerstone of collaboration, and all communications for team-based activities in the Software Engineering course must adhere to the below guidelines for professionalism. To be clear, we recognize two platforms for our official communication: Discord and GitHub. That having been said, the below guidelines ought to be followed for any in-person communication pertaining to or within the classroom.
```

Comment:
Maybe emphasize how important communication will be considering teams won't always be the same